### PR TITLE
fix(DB/BlackwingLair): Added missing spawns in Halls of Strife. Original author: Killyana (TrinityCore).

### LINKED ISSUE #10418: [BWL] Halls of Strife spawned creatures issues
### Current Behaviour

There are drakonids in the halls of strife
There are few whelps in the halls of strife
There are no taskmasters in the halls of strife
There are Death Talon Wyrmkin

### Expected Blizzlike Behaviour

There should be no drakonids in the halls of strife
there should be more whelps
there should be taskmasters
There should be no Death talon wyrmkin

Overal, the halls are very empty, it should be busy. See source;

### Source

https://youtu.be/oQUFxe47CKw?t=694

### Steps to reproduce the problem

.go c id 12468

### AC rev. hash/commit
fb8a38c7b3067414a8aaa4450756cc9f9143239a

### Operating system

Linux

### Custom changes or Modules

- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-progression-system](https://github.com/azerothcore/mod-progression-system)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zone-debuff](https://github.com/55Honey/Acore_ZoneDebuff)

### PR DESCRIPTION
**Pull request by UltraNix**
Fixed #10418

## Issues Addressed:

- Closes #10418

## Tests Performed:

- Not tested.

## How to Test the Changes:

`.go c id 12468`

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by UltraNix** [comment]
@Kitzunu Done.

File: data/sql/updates/pending_db_world/rev_1643565173013696100.sql [hunk 1]
```diff
@@ -0,0 +1,215 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1643565173013696100');
+
+DELETE FROM `creature_addon` WHERE `guid` IN (SELECT `guid` FROM `creature` WHERE `id1` IN (14262,14263,14264,14024,12458,12465));
+DELETE FROM `creature_formations` WHERE `memberGUID` IN (SELECT `guid` FROM `creature` WHERE `id1` IN (14262,14263,14264,14024,12458,12465));
+DELETE FROM `linked_respawn` WHERE `linkType`=0 AND `linkedGUID` IN (SELECT `guid` FROM `creature` WHERE `id1` IN (14262,14263,14264,14024,12458,12465));
+
+SET @CGUID := 2000009;
+DELETE FROM `creature` WHERE `id1` IN (14262,14263,14264,14024,12458,12465);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1643565173013696100.sql:8]: need to delete guid too, it is the key.

**Comment by FrancescoBorzi** [comment]
@UltraNix build failing

File: data/sql/updates/pending_db_world/rev_1643565173013696100.sql (hunk 1 above)
Comment (UltraNix) [reply data/sql/updates/pending_db_world/rev_1643565173013696100.sql:8]: No - delete by id is enough. It will delete entries with these guids.

**Comment by UltraNix** [comment]
@FrancescoBorzi IT's failing about SQL that is fine. I don't know why. These entries in `linked_respawn` are gone by 
```DELETE FROM `linked_respawn` WHERE `linkType`=0 AND `linkedGUID` IN (SELECT `guid` FROM `creature` WHERE `id1` IN (14262,14263,14264,14024,12458,12465));```
I believe it's because that line has been added later.

**Comment by UltraNix** [comment]
@FrancescoBorzi IT's failing about SQL that is fine. I don't know why. These entries in `linked_respawn` are gone by 
```DELETE FROM `linked_respawn` WHERE `linkType`=0 AND `linkedGUID` IN (SELECT `guid` FROM `creature` WHERE `id1` IN (14262,14263,14264,14024,12458,12465));```
I believe it's because that line has been added later.

**Comment by FrancescoBorzi** [comment]
@UltraNix if you confirm this can be merged safely, I will merge it.

**Comment by UltraNix** [comment]
@FrancescoBorzi Yes - you can merge it.

**Comment by FrancescoBorzi** [comment]
> @FrancescoBorzi Yes - you can merge it.

@UltraNix that did not work.... it is now failing on master too, can you please check it? we should quickly fix it or all PRs are going to fail because of that

**Pull request merged**